aidial_adapter_vertexai/universal_api/errors.py

```diff
@@ -0,0 +1,23 @@
+from typing import Mapping
+
+from aidial_sdk.chat_completion import Choice, Status
+from aiohttp import hdrs
+
+from aidial_adapter_vertexai.llm.exceptions import UserError
+
+
+def report_user_error(
```

Comment:
Is this function used anywhere?

Reply:
Removed along with a few other piece of dead code.